dgraphtest/local_cluster.go

```diff
@@ -626,6 +702,32 @@ func (c *LocalCluster) Client() (*GrpcClient, func(), error) {
 	return &GrpcClient{Dgraph: client}, cleanup, nil
 }
 
+// Client returns a grpc client that can talk to any Alpha in the cluster
+func (c *LocalCluster) ClientForAlpha(id int) (*GrpcClient, func(), error) {
```

Comment:
We should define this function such that existing (*LocalCluster).Client function can use it too. Feels duplicated code.

Reply:
Defined one for `conn`, clients are little high level to be able to use one for the other. Let me know if the modified piece is an ok abstraction.